src/inference/dev_api/openvino/runtime/internal_properties.hpp

```diff
@@ -29,6 +29,13 @@ static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> sup
  */
 static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> caching_properties{"CACHING_PROPERTIES"};
 
+/**
+ * @brief Read-only property to get a std::vector<PropertyName> of properties
+ * which should affect the loading time from cache
+ * @ingroup ov_dev_api_plugin_api
+ */
+static constexpr Property<std::vector<PropertyName>, PropertyMutability::RO> caching_with_mmap{"CACHING_WITH_MMAP"};
```

Comment:
Do we really need new property? I suppose user won't try to `read_model` with mmap, but `import_model` w/o mmap. If there are some objections when `mmap` not suitable, it will be disabled for both cases, so only one property should be enough

Reply:
Not all the plugins support reading the cache entry via the mmap, so this flag distinguishes such plugins. That is not the same as model reading.

Reply:
OK, got the idea